include/rviz_visual_tools/rviz_visual_tools.hpp

```diff
@@ -1075,18 +1094,27 @@ class RvizVisualTools
     psychedelic_mode_ = psychedelic_mode;
   }
 
+  // TODO(mlautman): Uncomment once https://github.com/ros2/rclcpp/issues/520 is addressed.
+  //                 Currently there is no way to spin without an executor so the remote
+  //                 control is not usable until this feature is complete.
```

Comment:
we can't make our own while loop with regular ol' timer?

Reply:
If we try to spin then we will get an exception `what(): Node has already been added to an executor.`